compiler/src/dotty/tools/repl/terminal/Utils.scala

```diff
@@ -178,8 +178,8 @@ object FrontEndUtils {
   def transpose[A](xs: List[List[A]]): List[List[A]] = {
     @tailrec def transpose(xs: List[List[A]], result: List[List[A]]): List[List[A]] = {
       xs.filter(_.nonEmpty) match {
-        case Nil =>  result
-        case ys: List[List[A]] => transpose(ys.map(_.tail), ys.map(_.head) :: result)
```

Comment:
This look like a bug. Scalac doesn't report a warning

Reply:
You are right, we already know that the list is a list of list of A. There is no runtime check needed.

Reply:
It might due to #3208

Reply:
Probably not, this is just a warning about erasure.

Reply:
I created #3323

Reply:
Though I would merge this change as the `: List[List[A]]` is absolutely useless.